packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository_bulk_delete_internal_types.ts

```diff
@@ -12,7 +12,7 @@ import type {
   ErrorCause,
 } from '@elastic/elasticsearch/lib/api/types';
 import type { estypes, TransportResult } from '@elastic/elasticsearch';
-import type { Either } from './internal_utils';
+import type { Either } from './apis/utils';
 import type { DeleteLegacyUrlAliasesParams } from './legacy_url_aliases';
 
 /**
```

Comment:
With the new structure, these should probably move somewhere else. Now that the "rules" are less strict, we could move them to `core-saved-objects-api-server` or to `base`.
Alternatively, just add them to the file that needs them.

Reply:
Yeah, I agree that our `Either` utils could be moved elsewhere. Although it's only used within the `api-server-internal` package (and will likely stay that way given we're using `fp-ts` in other parts of the code like the migration).

Ihmo the next step would probably be to delete those and using `fp-ts` too.

For the other utils thats harder. Utils are always a mess. Some are using only by one file, some are used by multiple APIs....